.devcontainer/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && \
 COPY . /app/
 
 # Expose port 49152
-EXPOSE 49152
+# EXPOSE 49152
```

Comment:
Can we delete this commented out part now?

Reply:
yes